angular/src/lib/tooltip/tooltip.directive.ts

```diff
@@ -42,16 +42,22 @@ export class TooltipDirective implements OnDestroy {
  /** @prop Sets the offset of the tooltip from the host */
   @Input() offset: number = 5;
 
-   /** @prop Sets the maxwidth of the tooltip */
-   @Input() maxWidth: number = 200;
+  /** @prop Sets the maxwidth of the tooltip */
+  @Input() maxWidth: number = 200;
+
+  /** @prop Allows tooltip to stay open until you click outside | false */
+  @Input() closeOnClick: boolean = false;
```

Comment:
Should this only be a prop on the Popover? We need to define the UX differences or similarities between Popover and Tooltip

Reply:
I would think both should have the capability

Reply:
if so we need a dark popover